src/main/java/com/github/manolo8/darkbot/utils/login/LoginUtils.java

```diff
@@ -81,21 +84,32 @@ public static LoginData performAutoLogin(StartupParams params) {
         loginData.setCredentials(username, password);
 
         try {
-            System.out.println("Auto logging in (1/2)");
-            usernameLogin(loginData);
+            if(isSidAutoLogin) loginData.setSid(sid, server + ".darkorbit.com");
+            else {
+                System.out.println("Auto logging in (1/2)");
+                usernameLogin(loginData);
+            }
             System.out.println("Loading spacemap (2/2)");
             findPreloader(loginData);
         } catch (IOException e) {
             System.err.println("IOException trying to perform auto login, servers may be down");
             e.printStackTrace();
         } catch (WrongCredentialsException e) {
+            if(isSidAutoLogin) {
```

Comment:
```suggestion
            if (isSidAutoLogin) {
```
Small formatting change